src/pages/Dashboard/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -37,9 +37,8 @@ export const DashboardHeader = memo(({ tabComponent }: { tabComponent?: React.Re
   const location = useLocation()
   const activeRef = useRef<HTMLButtonElement | null>(null)
   const containerRef = useRef<HTMLDivElement | null>(null)
-  const portfolioTotalUserCurrencyBalance = useAppSelector(
-    selectTotalPortfolioBalanceIncludeStakingUserCurrency,
```

Comment:
This selector is not consumed anywhere anymore, remove it 🙏

Reply:
eyy nice catch, bye bye she goes 👋🏽 

b7e5635929